identity/src/androidMain/kotlin/com/android/identity/device/DeviceCheckAndroid.kt

```diff
@@ -0,0 +1,46 @@
+package com.android.identity.device
```

Comment:
This file should be called `DeviceCheck.android.kt` and the iOS one should be called `DeviceCheck.ios.kt` (not all files in the project currently follow this convention)

Reply:
OK